builder/dockerfile/internals.go

```diff
@@ -504,6 +504,23 @@ func (b *Builder) create() (*daemon.Container, error) {
 	}
 	b.runConfig.Image = b.image
 
+	var mounts []string
+	potentialMounts := []string{
+		"/usr/bin/qemu-arm-static",
+		"/usr/bin/qemu-aarch64-static",
+	}
+
+	for _, mount := range potentialMounts {
+		if _, err := os.Stat(mount); err == nil {
+			toAdd := fmt.Sprintf("%s:%s:ro", mount, mount)
+			mounts = append(mounts, toAdd)
+			fmt.Fprintf(b.Stdout, " ---> Adding bind mount during build: %s\n", toAdd)
```

Comment:
I think we'd prefer these logs to be in docker's logs, not in the build output. You can use `logrus.Debugf` or some other log level. See the lines above